pkg/utils/sys_schema.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ReneKroon/ttlcache/v2"
+	"github.com/jinzhu/gorm"
+)
+
+const (
+	cacheTTL = 1 * time.Minute
+)
+
+type SysSchema struct {
+	ttl *ttlcache.Cache
```

Comment:
I think calling `cache` or `ttlCache` instead of `ttl` is better. simply because, it is a cache, while TTL is a feature of the cache. It is not a ttl. (think about it, what is "this thing is a time-to-live"? It should be "this thing is a cache".)